packages/easy-coding-standard/src/Console/Output/ConsoleOutputFormatter.php

```diff
@@ -161,7 +171,7 @@ private function printErrorMessageFromErrorCounts(int $errorCount, int $fileDiff
             '%s%d %s fixable! Just add "--fix" to console command and rerun to apply.',
             $errorCount !== 0 ? 'Good news is that ' : '',
             $fileDiffsCount,
-            $fileDiffsCount === 1 ? 'file is' : 'files are'
+            $fileDiffsCount === 1 ? 'error is' : 'errors are'
```

Comment:
here file diff, single file can have many diff error, eg in single markdown file